stdlib/public/core/ArrayShared.swift

```diff
@@ -143,7 +143,7 @@ internal func _expectEnd<C: Collection>(of s: C, is i: C.Index) {
 
 @inlinable
 internal func _growArrayCapacity(_ capacity: Int) -> Int {
-  return capacity * 2
+  return Int(Double(capacity) * 1.7)
```

Comment:
`capacity * 17 / 10` is probably more efficient than round-tripping through the FPU. The constant multiply-divide can be replaced with a magic multiplication.

Reply:
It doesn't matter much, except on cores without floating-point. Using 1 + 3/4 would be better, and basically equivalent. Also need to round-up (or special-case `capacity` == 1). Personally, I would probably use:
~~~~
return capacity + (capacity + 3)/4
~~~~

Reply:
> Also need to round-up (or special-case capacity == 1). 
🤦🏻‍♂️

Reply:
If it's any consolation, @natecook1000 managed to have exactly the same bug. Or as he put it, "implemented fixed-length arrays."